src/main/java/org/broadinstitute/hellbender/tools/spark/sv/sga/ContigAligner.java

```diff
@@ -1,13 +1,18 @@
-package org.broadinstitute.hellbender.tools.spark.sv;
+package org.broadinstitute.hellbender.tools.spark.sv.sga;
```

Comment:
This is going to `sga` because it's no longer used in the current, fermi-lite pathway, right? If so it might be good to add a comment and direct people where to go to find where the modern pipeline does BWA alignment.